src/woo_search/search_index/api/serializers/publications.py

```diff
@@ -51,12 +51,26 @@ class DocumentSerializer(serializers.Serializer):
         )
     )
     identifier = serializers.CharField(
-        help_text=_("The (primary) unique identifier."),
+        help_text=_("**DEPRECATED** The (primary) unique identifier."),
         max_length=255,
         required=False,
         allow_blank=True,
         default="",
     )
```

Comment:
Deprecation is done with the `drf-spectacular` tooling, OAS has a mechanism for that.